src/backend/marsha/core/tests/test_models_video.py

```diff
@@ -93,3 +96,53 @@ def test_models_video_is_ready_to_show(self):
             video = VideoFactory(live_state=live_choice[0], live_type=RAW)
 
             self.assertEqual(video.is_ready_to_show, True)
+
+    def test_models_video_is_scheduled_none(self):
+        """Checks that with a starting_at at None video it's not in the scheduled mode."""
+        # With no starting_date, video is not scheduled
+        video = VideoFactory(starting_at=None)
+        self.assertFalse(video.is_scheduled)
+        self.assertEqual(video.live_state, None)
+
+    def test_models_video_is_scheduled_with_state(self):
+        """The property is_scheduled is set according to starting_at and live_state."""
+        video = VideoFactory(starting_at="2100-01-01 10:00:00")
```

Comment:
This is not how you should set `starting_at`. You must pass it a timzone aware datetime.
In this example you could set it like this:
```suggestion
        video = VideoFactory(starting_at=timezone.now() + timedelta(hours=1))
```

This will allow you to remove your timezone squashings in the model :wink: